source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,348 @@
+.. _realm-sync-migrate-modes:
+
+=========================
+Migrate Device Sync Modes
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+Atlas App Services Apps can have one of two Device Sync modes: Partition-Based Sync
+and Flexible Sync. Partition-Based Sync is an older mode and you should consider
+migrating to Flexible Sync.
+
+Migrating an App that uses Partition-Based Sync to Flexible Sync is an automatic
```

Comment:
I think "app" should be lowercase here.

Reply:
And are we talking about a client app or AAS app?

Reply:
Ah, good point. I'll make this clearer.